frontend/public/components/modals/cluster-update-modal.tsx

```diff
@@ -88,11 +93,13 @@ const ClusterUpdateModal = withHandlePromise((props: ClusterUpdateModalProps) =>
     .filter((mcp) => !isMCPMaster(mcp))
     .sort(sortMCPsByCreationTimestamp);
   const pausedMCPs = pauseableMCPs.filter((mcp) => isMCPPaused(mcp));
+  const desiredRecommendedUpdate = _.find(availableSortedUpdates, { version: desiredVersion });
+  const desiredNotRecommendedUpdate = _.find(notRecommendedSortedUpdates, {
+    release: { version: desiredVersion },
+  });
   const submit: React.FormEventHandler<HTMLFormElement> = (e) => {
     e.preventDefault();
-    const available = getAvailableClusterUpdates(cv);
-    const desired = _.find(available, { version: desiredVersion });
-    if (!desired) {
+    if (!desiredRecommendedUpdate && !desiredNotRecommendedUpdate) {
       setError(
         `Version ${desiredVersion} not found among the available updates. Select another version.`,
```

Comment:
nit: `oc adm upgrade ...` has [a number of error strings][1] for different cases (the user wanted a recommended update, but there were none.  The user wanted a recommended update, but the version they selected is not among them.  The user wanted a supported update, regardless of whether it was recommended, but... , etc., etc.).  I dunno if it's worth building all that out here or not, but this particular message is using "available updates", while the `!desiredRecommendedUpdate && !desiredNotRecommendedUpdate` guard sounds more like `the update is not one of the possible targets` or `the update is not among supported updates` or something like that.


[1]: https://github.com/openshift/oc/blob/9b7fe72adeadeb1f950735eecaaa46a56694900e/pkg/cli/admin/upgrade/upgrade.go#L282-L309